src/main/sensors/voltage.c

```diff
@@ -89,7 +90,10 @@ const uint8_t supportedVoltageMeterCount = ARRAYLEN(voltageMeterIds);
 void voltageMeterReset(voltageMeter_t *meter)
 {
     meter->displayFiltered = 0;
+    meter->prevDisplayFiltered = 0;
+    meter->prevDisplayTime = 0;
```

Comment:
```suggestion
    meter->prevDisplayFilteredTime = 0;
```

Just maybe .. but  display + time is quite misleading combination

Reply:
changed to the better name